### DISCUSSION TITLE
How to config keystore path to Native build Quarkus with AWS

### DISCUSSION THREAD

**Discussion by LSZenerato** [quarkusio/quarkus#39113] [Q&A]
Hello!

My project needs to connect do DocumentDB that needs a RDS key to works properly, when i deploy my native quarkus build at AWS and run i get a error due to this missing rds key config. I tryed many ways but my config at application.properties never works.

My question is how to configure corretly de keystore ate my application.properties to work with a native build in AWS. Locally i can use my created keystore inside target/classes/keystore but in AWS throws **NoSuchFileException**, im preety sure my problem is my application.properties configs, but i didnt find any example without docker to use.

**Here is my config:**

- quarkus.native.additional-build-args= -Djavax.net.ssl.trustStore=target/classes/rds_keystore, -Djavax.net.ssl.trustStorePassword=changeit
- quarkus.http.ssl.certificate.key-store-file-type=JKS
- quarkus.http.ssl.certificate.key-store-file=target/classes/rds_keystore
- quarkus.http.ssl.certificate.key-store-password=changeit

Im new to Quarkus and not an expert at Java, how should i config my keystore to use with native mode? I tried the oficial Quarkus doc but doesnt seem to be complete and dont work as i needed.

Ty for help.

[a comment by an automated account is not shown]

**Comment by zakkak** [comment]
Hi @LSZenerato did you check https://quarkus.io/guides/native-and-ssl#the-truststore-path?

> Here is my config:
>
>    quarkus.native.additional-build-args= -Djavax.net.ssl.trustStore=target/classes/rds_keystore, -Djavax.net.ssl.trustStorePassword=changeit

It looks like you are missing some `-J`s, it should look like:

```
quarkus.native.additional-build-args=-J-Djavax.net.ssl.trustStore=target/classes/rds_keystore,-J-Djavax.net.ssl.trustStorePassword=changeit
```

Also please beware of https://github.com/quarkusio/quarkus/issues/13752 since you are using a relative path.
